improver_tests/acceptance/SHA256SUMS

```diff
@@ -633,73 +633,73 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
+c7cf5b8141df473b9854008aa25a9bdbdb162f840d6a829a46fbfccafa57d749  ./wxcode-modal/gridded_input/20201209T1300Z-weather_symbols-PT01H.nc
+bc1022649e1b0ace39d9da1f3fbca03be855a6ec785e4487b584139e92d7fb59  ./wxcode-modal/gridded_input/20201209T1400Z-weather_symbols-PT01H.nc
+724a98e9d311546af88b68796b4ba40968183b06ffd4688a0a24e1b6299864b1  ./wxcode-modal/gridded_input/20201209T1500Z-weather_symbols-PT01H.nc
+fd4ee760bee0077bc4abf0e4be0a8703c43e10097fa8de08b1abf9a3ea4a780b  ./wxcode-modal/gridded_input/20201209T1600Z-weather_symbols-PT01H.nc
+5d2fa6f476337c2c3783b77d52615652f269174d3b8e804e7c6f931c3e7df4dd  ./wxcode-modal/gridded_input/20201209T1700Z-weather_symbols-PT01H.nc
+8d150f86a12e4b914e7f2ee269e643022db664d26c2539927d83f4424706cb3e  ./wxcode-modal/gridded_input/20201209T1800Z-weather_symbols-PT01H.nc
+a37d508c6c356325b1c31c245726aa5459484c4fb34fb2fbf309b10265d24712  ./wxcode-modal/gridded_input/kgo.nc
+94a8758ca265acf625a21f4f9c6a246181b7b51e35710f312675d22b394f4d54  ./wxcode-modal/gridded_ties/20201209T0700Z-weather_symbols-PT01H.nc
+0dbf6702561e52bcd97a86ef3d954e353a5218b78a1376792144eeb0a673a0c8  ./wxcode-modal/gridded_ties/20201209T0800Z-weather_symbols-PT01H.nc
+08881255e7b71679aa1eda6e7bec8b85ceba5adc3547ae169e192abfc3238c39  ./wxcode-modal/gridded_ties/20201209T0900Z-weather_symbols-PT01H.nc
+fc4a6da438ac558080c11413de21cd9943f356795d6f145f8e43e9ca290d1412  ./wxcode-modal/gridded_ties/20201209T1000Z-weather_symbols-PT01H.nc
+ed07ba61392f465d5890a06c6ad3cee9d9adf31ab03ed365e1d9771d005d646c  ./wxcode-modal/gridded_ties/20201209T1100Z-weather_symbols-PT01H.nc
+350f0754f7bb247a0a81bb81d05f3e69da4d8d768d8269c82f6fef73713b5b06  ./wxcode-modal/gridded_ties/20201209T1200Z-weather_symbols-PT01H.nc
+e8dc643ec6446d495a96d4dda6104b958d08137c15c3e832bb397a50229062eb  ./wxcode-modal/gridded_ties/20201209T1300Z-weather_symbols-PT01H.nc
+77375be1e8258f0b641ea6475c7739beec64252dc66ae4216a5cc02a3f01cd6e  ./wxcode-modal/gridded_ties/20201209T1400Z-weather_symbols-PT01H.nc
+148cdbc9282a8fb44e4793144852757ee36c5cd63d6efebfcb63337186e96a2f  ./wxcode-modal/gridded_ties/20201209T1500Z-weather_symbols-PT01H.nc
+44c510792f566329597239006b43ea736b5faa1a884202fd523aba31ebc4f445  ./wxcode-modal/gridded_ties/20201209T1600Z-weather_symbols-PT01H.nc
+182211dba9b8d18c2d48dce89f2871854ae9baa9457bc9cfad16c1cea80faf09  ./wxcode-modal/gridded_ties/20201209T1700Z-weather_symbols-PT01H.nc
+2ed494728e73e8f740680ae090504770ee6b60131d543698c640db5ed4e08598  ./wxcode-modal/gridded_ties/20201209T1800Z-weather_symbols-PT01H.nc
+fab89f125fe7b016a35fcbf52cd9b713cfa28c7009b90150f289f47d7bea6b12  ./wxcode-modal/gridded_ties/kgo.nc
+9fbcfc0e0c3f6af0d07690c6be12e2f641123168af0c7e165b8e0ec6a2ca6f98  ./wxcode-modal/single_input/20201210T0000Z-weather_symbols-PT01H.nc
+4e52a42de445064a987da6f93b7e888c0a8ca974063a4d9cf9818a57d9b60367  ./wxcode-modal/single_input/kgo.nc
```

Comment:
For both of these files, the `wmo_id` and `met_office_site_id` units are `1` in both master and the acceptance test branch but I think it should be `no_unit`.

Reply:
I've recreated the KGO (the associated tests were failing and I hadn't noticed). These units look correct now.